library/ssl_tls13_keys.c

```diff
@@ -1510,4 +1511,56 @@ int mbedtls_ssl_tls13_generate_application_keys(
     return( ret );
 }
 
+int mbedtls_ssl_tls13_compute_handshake_transform( mbedtls_ssl_context *ssl )
+{
+    int ret;
```

Comment:
Should we give the initial value "int ret = MBEDTLS_ERR_ERROR_CORRUPTION_DETECTED;"?